modules/500-upmeter/images/upmeter/pkg/agent/sender/client.go

```diff
@@ -142,15 +144,17 @@ func createHttpTransport(caPath string) (*http.Transport, error) {
 	caCertPool := x509.NewCertPool()
 	caCertPool.AppendCertsFromPEM(caCertBytes)
 
-	tr := &http.Transport{
-		TLSClientConfig: &tls.Config{
-			RootCAs: caCertPool,
-		},
-	}
+	tr.TLSClientConfig = &tls.Config{RootCAs: caCertPool}
 
 	return tr, nil
 }
 
+func newTransport(timeout time.Duration) *http.Transport {
```

Comment:
It is unclear that the timeout argument is actually a ResponseHeaderTimeout.

Reply:
Let it be just "a transport timeout"! it is reused in multiple places.